### DISCUSSION TITLE
Using `src="..."` for external images

### DISCUSSION THREAD

**Discussion by adevade** [spatie/statamic-responsive-images#223] [Q&A]
Hello!

I don't think it's possible to use this addon for external image links, correct? We have some images from the Instagram API that would be nice to easily pass in to this tag and have it generate variants.

It's possible with the `{{ glide }}` tag in Statamic: https://statamic.dev/tags/glide#external-url.

Is this something that is possible, and would you consider it a good feature? If so, I could look into a PR when I have a little more time. 

Thanks!

**Comment by ncla** [comment]
Yeah, you could, but this seems wrong from performance viewpoint. Is Glide smart enough to cache the remote image? Does it send request once for one external asset URL or one for every Glide manipulation?

I would also make a local copy of these Instagram images, as external images could vanish, but I may not know your exact usage and this maybe okay for you regardless.

If you can write tests in the PR, feel free to send in the PR.
